parm/jcb-rdas/test/ci/jcb-rrfs_fv3jedi_2024052700_3denvar.yaml

```diff
@@ -74,6 +74,7 @@ atmosphere_background_time_iso: '2024-05-27T00:00:00Z'
 atmosphere_background_time_prefix: "20240527.000000."
 filename_core: fv_core.res.tile1.nc
 filename_trcr: fv_tracer.res.tile1.nc
+filename_phys: phy_data_prepdbz.nc
```

Comment:
A question. from my understanding this yaml will be used for run without dbz obs, right?  If so, why do we need this file here?